hack/vagrant/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.boot_timeout=600
+  config.vm.box = "bento/ubuntu-16.04"
```

Comment:
Can we use the official image? 
BTW, please offer the kernel version, since the containerd requires the  higher kernel version. If you can offer the information, it maybe help contributor to diagnose.